theatre/studio/src/uiComponents/simpleContextMenu/getCopyPasteKeyframesItem.ts

```diff
@@ -0,0 +1,107 @@
+import {val} from '@theatre/dataverse'
+import type {Keyframe} from '@theatre/core/projects/store/types/SheetState_Historic'
+import type {DopeSheetSelection} from '@theatre/studio/panels/SequenceEditorPanel/layout/layout'
+import getStudio from '@theatre/studio/getStudio'
+import type {SequenceEditorTree_PrimitiveProp} from '@theatre/studio/panels/SequenceEditorPanel/layout/tree'
+import {getCopiedKeyframes} from '@theatre/studio/selectors'
+import type {IContextMenuItem} from './useContextMenu'
+
+export const getPasteKeyframesItem = (
```

Comment:
the word `Item` is a bit non-descriptive. Perhaps `ContextMenuItem` works better? Or `getContextMenuItemForPasteKeyframes`? It's long, but descriptive.